graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/GqlJcrNodeInput.java

```diff
@@ -56,17 +56,20 @@
 
     private String name;
     private String primaryNodeType;
+    private Boolean useAvailableNodeName;
     private Collection<String> mixins;
     private Collection<GqlJcrPropertyInput> properties;
     private Collection<GqlJcrNodeInput> children;
 
     public GqlJcrNodeInput(@GraphQLName("name") @GraphQLNonNull @GraphQLDescription("The name of the node to create") String name,
-                           @GraphQLName("primaryNodeType") @GraphQLNonNull @GraphQLDescription("The primary node type of the node to create") String primaryNodeType,
+                           @GraphQLName("primaryNodeType") @GraphQLDescription("The primary node type of the node to create") String primaryNodeType,
```

Comment:
Why not @GraphQLNonNull anymore ?

Reply:
it was a typo, thanks for notify it!
fixed by amending initial commit